addons/im_livechat/static/tests/translation.test.js

```diff
@@ -27,3 +27,22 @@ test("message translation in livechat", async () => {
     await click("[title='Expand']");
     await contains("[title='Translate']");
 });
+
+test("message translation in livechat (agent is not member)", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        anonymous_name: "Visitor",
+        channel_type: "livechat",
+        channel_member_ids: [Command.create({ partner_id: serverState.publicPartnerId })],
```

Comment:
This doesn't match real world behavior: we do not add public users to channel anymore.